modules/jolt_physics/jolt_globals.cpp

```diff
@@ -53,6 +53,9 @@ void *jolt_realloc(void *p_mem, size_t p_old_size, size_t p_new_size) {
 }
 
 void jolt_free(void *p_mem) {
+	if (p_mem == nullptr) {
+		return;
+	}
 	Memory::free_static(p_mem);
```

Comment:
```suggestion
	if (unlikely(p_mem == nullptr)) {
		return;
	}
	Memory::free_static(p_mem);
```

I doubt this makes any difference at all, but Godot uses branch prediction hints all over the place, so let's add them here as well I guess.